src/app/components/blog/blog-preview-card/blog-preview-card.component.ts

```diff
@@ -8,10 +8,16 @@ import { Component, Input, OnInit } from '@angular/core';
 export class BlogPreviewCardComponent implements OnInit {
   @Input() post;
   date;
-  constructor() {}
+  constructor() { }
 
-   ngOnInit() { 
-    this.date = new Date(this.post.published).toDateString();
+  ngOnInit() {
+    this.createDisplayDate()
+  }
+
+  createDisplayDate() {
```

Comment:
Should use a built in Angular pipe, or make your own pipe for this kind of thing.